src/clj/game/core/moving.clj

```diff
@@ -510,6 +514,24 @@
         (when (in-hand? moved-b) (add-to-currently-drawing state b-side moved-b)))
       [(get-card state moved-a) (get-card state moved-b)])))
 
+(defn swap-cards-async
```

Comment:
I don't think this is being used anywhere.

Reply:
Currently being used in tatu-bola to make a teia trigger when it's swapped out (I also added a test for it). 
If players find more cases where we need to use it, I can add those in later.